packages/typescript/test/ts_project/js_library/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "files": ["b.ts"],
+    "compilerOptions": {
```

Comment:
this shouldn't be needed here since the .d.ts was in the source tree right?

Reply:
oh, I see, js_library puts all its outputs into the out dir, so even though there's a .d.ts in the sources it's not the one the ts_project rule will see.
that's pretty confusing for users :(